sdk/java/pulumi/src/main/java/com/pulumi/deployment/MockRunner.java

```diff
@@ -0,0 +1,39 @@
+package com.pulumi.deployment;
+
+import com.pulumi.core.Output;
+import com.pulumi.deployment.internal.Runner;
+import com.pulumi.resources.Stack;
+import com.pulumi.resources.StackOptions;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Supplier;
+
+public class MockRunner implements Runner {
```

Comment:
not sure if we'll need this class in the end, but right now it does exactly what a mockito mock would do

Reply:
Great that you've come to the same conclusions independently, this gives me confidence in this decision.

Yes, I totally agree we should make all mocks except the `Mocks` (monitor mocks interface and maybe a default impl `EmptyMocks`) internal.